draft/draft-ietf-bfd-stability.xml

```diff
@@ -219,7 +219,7 @@
     <section anchor="null-auth-type" title="NULL Auth Type">
       <t>
 	The NULL Authentication Type, defined in this document, can be
-	used to provide a meticulously increasing sequence number for
+	used to provide a monotonically increasing sequence number for
```

Comment:
This is wrong, although the property we want is also monotonic.  Monotonic only guarantees that it increases.  The meticulous property means it's always x++.

If you don't have that property, you can't find sequence gaps.